Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -77,15 +93,17 @@ public class ApolloCodegen {
   ///     If `nil`, the current working directory of the executing process will be used.
   public static func build(
     with configuration: ApolloCodegenConfiguration,
-    withRootURL rootURL: URL? = nil
+    withRootURL rootURL: URL? = nil,
+    buildOptions: CodeGenerationBuildOptions = [.code]
   ) throws {
     try build(with: configuration, rootURL: rootURL)
   }
 
   internal static func build(
     with configuration: ApolloCodegenConfiguration,
     rootURL: URL? = nil,
-    fileManager: ApolloFileManager = .default
+    fileManager: ApolloFileManager = .default,
+    buildOptions: CodeGenerationBuildOptions = [.code]
```

Comment:
I usually only add the default value to the public initializers in situations like this.